src/CoreBundle/private/modules/MTHeader/views/breadcrumb.view.php

```diff
@@ -1,5 +1,4 @@
 <?php
-if (isset($data['oUser'])) {
```

Comment:
Why is this deleted?

Reply:
because we can always show the breadcrumb - it is not part of the login page anymore... (and we no longer have oUser)